lua/telescope/builtin/internal.lua

```diff
@@ -926,6 +926,30 @@ internal.tagstack = function(opts)
     }):find()
 end
 
+internal.jumplist = function(opts)
+  opts = opts or {}
+  local jumplist = vim.api.nvim_eval('getjumplist()')[1]
```

Comment:
Just as an fyi, you should be able to use `vim.fn` here to call the `getjumplist()` function. `vim.fn` is the Neovim Lua api for calling Vimscript functions in Lua, so you could change that line to:
```lua
local jumplist = vim.fn.getjumplist()[1]
```